process/block/preprocess/intermediateResults.go

```diff
@@ -195,6 +193,13 @@ func (irp *intermediateResultsProcessor) SaveCurrentIntermediateTxToStorage() er
 	return nil
 }
 
+// CreateBlockStarted cleans the local cache map for processed/created intermediate transactions at this round
+func (irp *intermediateResultsProcessor) CreateBlockStarted() {
```

Comment:
did you started the code from Corco's branch ? there are some bugfixes from the smart contract processor.